packages/commons-server/test/suites/templating-helpers/helpers.spec.ts

```diff
@@ -199,6 +202,60 @@ describe('Template parser', () => {
     });
   });
 
+  describe('Helper: date', () => {
+    it('Should return an empty string if given the wrong amount of arguments', () => {
+      const parseResult = TemplateParser('{{date}}', {} as any, {} as any);
+      expect(parseResult).to.be.equal('');
+    });
+
+    it('Should return an empty string if given the wrong amount of arguments', () => {
+      const parseResult = TemplateParser(
+        "{{date '2022-01-01'}}",
+        {} as any,
+        {} as any
+      );
+
+      expect(parseResult).to.be.equal('');
+    });
+
+    it('Should return a date using a the default format', () => {
+      const parseResult = TemplateParser(
+        "{{date '2022-01-01' '2022-02-01'}}",
```

Comment:
Add a format to avoid failing tests due to different machine's locales

```suggestion
        "{{date '2022-01-01' '2022-02-01' 'YYYY'}}",
```